code/__DEFINES/components.dm

```diff
@@ -16,3 +16,4 @@
 #define COMSIG_PARENT_QDELETED "parent_qdeleted"				//before a datum's Destroy() is called: ()
 #define COMSIG_ATOM_ENTERED "atom_entered"						//from base of atom/Entered(): (atom/movable, atom)
 #define COMSIG_MOVABLE_CROSSED "movable_crossed"				//from base of atom/movable/Crossed(): (atom/movable)
+#define COMSIG_OPENTURF_ATTACKBY "turf_open_attackby"
```

Comment:
Why have this be turf specific? Surely other things will want to have attackby comsigs of their own, and if a nonturf is winding up with an archaeology component then they've got bigger problems

Reply:
so in other words you just want it to be a generic ATTACKBY signal?

Reply:
I added this exact signal in #29785 might wanna use that